packages/headless/src/features/result-preview/result-preview-state.ts

```diff
@@ -2,12 +2,14 @@ export interface ResultPreviewState {
   uniqueId: string;
   content: string;
   isLoading: boolean;
+  maximumFileSize: number;
 }
 
 export function getResultPreviewInitialState(): ResultPreviewState {
   return {
     uniqueId: '',
     content: '',
     isLoading: false,
+    maximumFileSize: 50,
```

Comment:
decided to make the default value 50 MB.

Reply:
I just choose it arbitrarily. I'm not sure what a good value would be. But if the default value is 0, doesn't that mean there will be no preview since it won't allow any documents with a size larger than 0 to have a quickview?